pymc/tests/test_distributions.py

```diff
@@ -2216,20 +2216,44 @@ def test_multinomial(self, n):
             lambda value, n, p: scipy.stats.multinomial.logpmf(value, n, p),
         )
 
-    def test_multinomial_invalid(self):
-        # Test non-scalar invalid parameters/values
-        value = np.array([[1, 2, 2], [4, 0, 1]])
-
-        invalid_dist = Multinomial.dist(n=5, p=[-1, 1, 1], size=2)
-        # TODO: Multinomial normalizes p, so it is impossible to trigger p checks
-        # with pytest.raises(ParameterValueError):
-        with does_not_raise():
-            pm.logp(invalid_dist, value).eval()
-
-        value[1] -= 1
+    def test_multinomial_invalid_value(self):
+        # Test passing non-scalar invalid parameters/values to an otherwise valid Multinomial, 
+        # evaluates to -inf 
+        value = np.array([[1, 2, 2], [3, -1, 0]])
         valid_dist = Multinomial.dist(n=5, p=np.ones(3) / 3)
         assert np.all(np.isfinite(pm.logp(valid_dist, value).eval()) == np.array([True, False]))
-
+    
+    def test_multinomial_negative_p(self):
+        # test passing a list/numpy with negative p raises an immediate error
+        with pytest.raises(ValueError, match = "[-1, 1, 1]"):
+            with Model() as model:
+                x = Multinomial("x", n = 5, p=[-1, 1, 1])
+    
+    def test_multinomial_p_not_normalized(self):
+        # test UserWarning is raised for p vals that sum to more than 1
+        # and normaliation is triggered 
+        with pytest.warns(UserWarning, match = "[5]"):
+            with pm.Model() as m:
+                x = pm.Multinomial('x', n=5, p=[1,1,1,1,1])
+        # test stored p-vals have been normalised 
+        assert np.isclose(m.x.owner.inputs[4].sum().eval(), 1.0)
+        
+    def test_multinomial_negative_p_symbolic(self):
+        # Passing symbolic negative p does not raise an immediate error, but evaluating 
+        # logp raises a ParameterValueError
+        with pytest.raises(ParameterValueError):  
+            value = np.array([[1, 1, 1]])
+            invalid_dist = pm.Multinomial.dist(n=1, p=at.as_tensor_variable([-1, 0.5, 0.5]))
+            pm.logp(invalid_dist, value).eval()
+            
+    def test_multinomial_nonnorm_p_symbolic(self):
```

Comment:
This one seems to have escaped

```suggestion
    def test_multinomial_p_not_normalized_symbolic(self):
```